README.md

```diff
@@ -1,2 +1,163 @@
-# docker-php
+# PHP-FPM
+
+[![Docker Stars](https://img.shields.io/docker/stars/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![Docker Pulls](https://img.shields.io/docker/pulls/spryker/php.svg)](https://store.docker.com/community/images/spryker/php)
+[![microbadger version](https://images.microbadger.com/badges/version/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own version badge on microbadger.com")
+[![microbadger image](https://images.microbadger.com/badges/image/spryker/php.svg)](https://microbadger.com/images/spryker/php "Get your own image badge on microbadger.com")
+
+# Description
+
 Extends official PHP Docker images with extensions and tools to be able to run Spryker on.
+
+* Based on: Official PHP images and `Alpine 3.9`
+* Users: `root`, `spryker`
+* Working directory: `/data`
+* Includes:
+  * [PHP extensions](#php-extensions)
+  * PostgreSQL client
+  * MySQL client
+  * CURL
+  * OpenSSH
+  * Composer
+  * `hirak/prestissimo`
+  * `aws/aws-sdk-php`
+
+> Note: Provided images require additional configuration for development, staging and production use.
+
+## Tags
```

Comment:
Are you sure you want to include list of tags here? That's a manual process to update this file each time new PHP version is released

Reply:
Yep. We will do that in Dockerfiles and in README.